translations/catroweb.en_AU.yaml

```diff
@@ -207,6 +207,9 @@ programs:
   editTranslation: Edit %language% translation
   translations: Translations
   createTranslation: Create translation
+  deletedComment: '**Deleted**'
+  replyLabel: Reply
+  repliesLabel: Replies
```

Comment:
you only have to change the base file
catroweb.en.yml.

All other files will be generated with Crowdin.